spec.md

```diff
@@ -595,6 +595,10 @@ message PluginCapability {
     Type type = 1;
   }
 
+  // Deprecated - Ability for a plugin to delcare online and offline
+  // controller expansion capabilities via PluginCapability
+  // is deprecated. A plugin may support either mode of operation
+  // without having to declare them in PluginCapability.
   message VolumeExpansion {
     enum Type {
       UNKNOWN = 0;
```

Comment:
Should we also remove the comments on `ONLINE = 1;` and `OFFLINE = 2;` below and replace them with `// Deprecated` + ` [deprecated=true]` annotation?